sdk/storage/azfile/internal/shared/shared.go

```diff
@@ -47,6 +47,8 @@ const (
 	// Inherit inherits permissions from the parent folder (default when creating files/folders)
 	DefaultFilePermissionString = "inherit"
 
+	DefaultFilePermissionFormat = "Sddl"
```

Comment:
Default value refers to "sddl" (complete lower case) in docs. Confirm if this is the case.